src/pages/solve-overview/template.html

```diff
@@ -24,10 +24,6 @@ <h3 v-html="topic.desc"></h3>
             <span class="rsTl-article-read"><strong>{{ featured.first.field_content_type | translate }}</strong> ({{ '@text minute ' + viewAction[featured.first.field_content_type_key] | translate({'@text': featured.first.field_content_length}) }})</span>
             <h1 class="rsTl-feature-title"><a class="rsTl-feature-titleLink track-cta" :href="featured.first.alias" v-html="featured.first.header"></a></h1>
             <p class="rsTl-feature-p" v-html="$options.filters.truncate(featured.first.body, 200)"></p>
-            <p class="rsTl-feature-p">
-              <span class="rsTl-feature-author" v-if="isSyndicatedItem(featured.first)" v-html="featured.first.field_attribution_name"></span>
-              <span class="rsTl-feature-author" v-else v-html="featured.first.field_author"></span> – {{ featured.first.field_published_date_formatted }}
-            </p>
```

Comment:
We don't want remove this whole thing.  We only need to remove the `v-if` portion.  By removing the `v-else` statement (and surrounding `<p>` tag), we're dropping functionality here.